lib/util/extract-sourcemap.js

```diff
@@ -0,0 +1,18 @@
+let { readFileSync } = require("fs");
+let regex = /sourceMappingURL=data:application\/json;base64,(\S+)/;
+
+// Extract the given attributes of a sourcemap as JSON from a given source file
+//
+// Exported as a global so it can be used with node -r
```

Comment:
* elsewhere we use indicative rather than imperative mood - i.e. I'd prefer "extracts"
* I believe we agreed not to use sentence case? at least `cli_harness.sh` doesn't seem to
* backticks around `node -r` would avoid ambiguity
* I still don't know whether it should read "source map" or "sourcemap" - I've only ever seen "source maps", which seems subtly different (it is Friday, 22:19 😂 🔫 )

Reply:
Yes to all of these things. Will do it alongside all other code changes.

[MDN says "source map"](https://developer.mozilla.org/en-US/docs/Tools/Debugger/How_to/Use_a_source_map)